src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/MålgruppeValidering.kt

```diff
@@ -27,4 +34,96 @@ object MålgruppeValidering {
             "målgruppe=$målgruppeType er ikke gyldig for $stønadstype"
         }
     }
+
```

Comment:
Er det ønskelig å kunna legge inn inngangsvilkår som ikke oppfyller aldersvilkåret? 
Eks at man validerer at et inngangsvilkår HELT oppfyller alder, eller ikke. Der valideringen sjekker at den et strengt ok, eller strengt ikke ok i perioden

Reply:
Yes, dette gir mening. Her kan vi kaste feilmelding for alle Vilkårsperioder som ikke oppfyller Aldersvilkår

Reply:
Ref din kommentar over @blommish: Snakket med Marie om dette. Det er ønskelig å tillate lagring av vilkårsperioder som ikke har gydlig aldersvilkår, på lik linje som med andre vilkår ("lønnet arbeid", f. eks)

Reply:
Åpnet opp for lagring av Målgrupper hvor Aldersvilkår ikke er gyldig: 6c24938c674ed7b8b4d24939f3692cf5c0492d86